src/main/java/frc/robot/RobotContainer.java

```diff
@@ -46,6 +59,11 @@ private void configureBindings() {
       driveSubsystem.setDrivePowers(driverController.getLeftY(), driverController.getRightY());
```

Comment:
wrap these in a `.setDefaultCommand(new InstantCommand( () -> ))` so the setters are colled continuously and not just once.